avatica/src/main/java/org/apache/calcite/avatica/proto/Requests.java

```diff
@@ -14,7 +14,6 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-
 // Generated by the protocol buffer compiler.  DO NOT EDIT!
```

Comment:
Ah, I see. The tool is native, thus there is no way to launch it without manual installation.